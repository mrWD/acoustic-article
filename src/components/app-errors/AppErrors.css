.AppErrors {
  margin-top: 0;
  margin-bottom: 0;
  padding-top: 60px;
  padding-bottom: 60px;
  font-size: 24px;
  text-align: center;
  color: #323232;
}