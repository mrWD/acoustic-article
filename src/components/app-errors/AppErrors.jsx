import React from 'react';
import PropTypes from 'prop-types';

import './AppErrors.css';

const AppErrors = props => (
  <p className="AppErrors">{props.errors}</p>
);

AppErrors.propTypes = {
  errors: PropTypes.string.isRequired,
};

AppErrors.defaultProps = {
  errors: '',
};

export default AppErrors;
