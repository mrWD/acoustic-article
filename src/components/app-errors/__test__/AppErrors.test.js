import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

import AppErrors from '../AppErrors';

configure({ adapter: new Adapter() });

describe('AppErrors', () => {
  test('renders without crashing', () => {
    shallow(<AppErrors />);
  });

  test('renders text', () => {
    const wrapper = shallow(<AppErrors errors="Error text" />);

    expect(wrapper.text()).toBe('Error text');
    expect(wrapper).toMatchSnapshot();
  });
});
