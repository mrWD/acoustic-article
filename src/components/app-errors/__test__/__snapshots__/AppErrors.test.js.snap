// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppErrors renders text 1`] = `
<p
  className="AppErrors"
>
  Error text
</p>
`;
